Sources/BrowserServicesKit/Autofill/AutofillUserScript+Email.swift

```diff
@@ -120,4 +126,19 @@ extension AutofillUserScript {
         }
     }
 
+    private struct DeviceCapabilities: Encodable {
```

Comment:
This will clash a little (semantically, at least) with the work the iOS folks are doing for [Autofill](https://github.com/duckduckgo/BrowserServicesKit/compare/elle/autofill-logins-ios#diff-56aef5a70fd2178a280a29fa7eddfd455d80ed3a92eaa07eddacfb465e2ce047R47), since they've defined a separate struct which implements the same type of feature. If this struct is just for email, what do you think about renaming it to `DeviceEmailCapabilities` just to separate it from that?

Reply:
@samsymons Good shout! Updated 👍 Hopefully we can align the two in the future